packages/codegen-ui/lib/generate-form-definition/form-to-component.ts

```diff
@@ -106,7 +106,10 @@ export const ctaButtonConfig = (): StudioComponentChild => {
         name: 'CancelButton',
         properties: {
           label: {
-            value: 'Cancel',
+            value:
+              formDefinition.buttons?.cancel?.visible && formDefinition.buttons?.cancel?.labelOverride
+                ? formDefinition.buttons?.cancel?.labelOverride
+                : 'Cancel',
```

Comment:
Can we have section above which renders the value with formDef and handles the default value as well?

Reply:
added resolveCtaLabels section to handle this